src/types/ReviewLink.ts

```diff
@@ -0,0 +1,4 @@
+export enum SEARCH_TYPE {
```

Comment:
Couldn't we re-use the one enum type we defined for the search form ?

Reply:
I think that one you are mentioning is for the Record type, I can't seem to find it, could you please show me which one ?

Reply:
Ok, never mind it's flag label (FlagLabel in SimpleSearch.ts).

Also, only constants are in UPPCASE+ SNAKE CASE. So use